sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -217,4 +217,38 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
       .where("location.id = :locationId", { locationId })
       .getRawOne();
   }
+
+  /**
+   * Get an institution's locations with designation statuses.
+   * @param institutionId institution id
+   * @returns InstitutionLocation[]
+   */
+  async getInstitutionLocationsWithDesignationStatus(
+    institutionId: number,
+  ): Promise<LocationWithDesigationStatus[]> {
```

Comment:
Please fix the typo on LocationWithDesigationStatus. I can see that it was copied from the previous PR which was also incorrect.